components/home/menu/shuttleBusSchedule/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import {
+  View, Text, ScrollView, SectionList
+} from 'react-native';
+import { ButtonGroup } from 'react-native-elements';
+import shuttleScheduleInformation from './shuttleScheduleService';
+import styles from './styles';
+
+const buttons = ['SGW', 'LOY'];
+export default class ShuttleSchedule extends Component {
+  constructor() {
+    super();
+    this.state = {
+      selectedButtonIndex: 0
+    };
+    this.updateButtonIndex = this.updateButtonIndex.bind(this);
+  }
+
+  /** The function will return the appropriate schedule.
+   * Button index is as follows: 0 -> SGW and 1->LOY
+   * @param {Number} selectedButtonIndex - index of the button, either 0 or 1
+   */
+  getShuttleCampusInformation(selectedButtonIndex) {
+    if (selectedButtonIndex === 1) { return shuttleScheduleInformation.Mon_Thu.LOY; }
```

Comment:
Might be worth looking into using enums for button indices.
This may be a suggestion to take with a grain of salt depending on how difficult it is to implement enums for react native.

Reply:
There is already some logic implemented with the index of the buttons, perhaps a later fix